package.json

```
@@ -1,39 +1,24 @@
 {
   "name": "hchiam.github.io",
-  "version": "4.0.0",
+  "version": "5.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "export": "next export",
-    "deploy": "next build && next export && npm run deploy:pre-deploy && npm run deploy:surge && npm run deploy:gh-pages",
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "deploy": "astro build && npm run deploy:pre-deploy && npm run deploy:gh-pages",
     "deploy:pre-deploy": "cp {.nojekyll,CNAME,manifest.webmanifest} out && git add . && git commit -m \"Deploy to gh-pages\" && git push && git status",
-    "deploy:surge": "surge out https://hchiam.surge.sh",
```

we still might want `deploy:surge` to go to https://hchiam-test.surge.sh
```suggestion
"deploy:surge": "surge out https://hchiam-test.surge.sh",
```